crates/sui-core/src/authority/authority_store_tables.rs

```diff
@@ -51,6 +51,7 @@ impl AuthorityPerpetualTablesOptions {
 
 /// AuthorityPerpetualTables contains data that must be preserved from one epoch to the next.
 #[derive(DBMapUtils)]
+// #[tidehunter]
```

Comment:
We could un-comment this line(maybe rename it to `tidehunter-supported`) and commit it, and then in the proc-macro we could check both this attribute **and** tidehunter feature enabled. This way one less thing to modify when we want to enable tidehunter

Reply:
Or, alternatively, maybe [cfg-attr](https://doc.rust-lang.org/reference/conditional-compilation.html#the-cfg_attr-attribute) can be used instead?

Reply:
we can iterate on top of it, but from what I remember, I didn't find a clean way to do it because some of the CI jobs run with all features enabled

Reply:
cfg-attr can have same conditions as you have with not(windows) and tidehunter?